app/javascript/lib/metrics-api.js

```diff
@@ -163,17 +163,24 @@ function getLabelsForElement(element) {
  * Log click on input by type, e.g. text, number, checkbox
  */
 function logClickInput(target) {
-  const domLabels = getLabelsForElement(target)
+  let props
+  if (target.type === 'radio') {
+    const id = 'id' in target ? target.id : ''
```

Comment:
I assume you tested this and so it's there for a reason, but I thought (element).id already returned empty string if no id was specified for the element

Reply:
Good catch, refined in f86f76a.  I used this construct yesterday for something else, then removed it, then reflexively used it here.